db/migrate/20160605140228_create_searches.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSearches < ActiveRecord::Migration
```

Comment:
more confidence in not changing timestamps due to this: http://stackoverflow.com/questions/36608063/merging-activerecord-migrations-out-of-order